app/assets/javascripts/discourse/components/d-editor.js.es6

```diff
@@ -892,7 +892,7 @@ export default Component.extend({
       }
     }
 
-    if (handled) {
+    if (handled || (!handled && canUpload)) {
```

Comment:
I think you can shorten that to

```javascript
if (handled || canUpload) {
```

Reply:
Yes we already had this discussion, I just think my version much more readable. Making both conditions explicit, instead of making the second one relying on the first one. But I'm fine changing it no problem.

Reply:
I have to side with @ZogStriP here - the negative makes me personally read it as much more complicated than it is.

Reply:
I agree with @ZogStriP & @eviltrout. We can add JS comment to prevent confusion.

Reply:
No a comment is not needed, all right let's go with this 👍